lib/modules/manager/types.ts

```diff
@@ -15,7 +15,9 @@ export interface ManagerData<T> {
 }
 
 export interface ExtractConfig {
-  aliases?: Record<string, string>;
+  registryUrls?: string[];
+  endpoint?: string;
```

Comment:
```suggestion
```

Seems wrong/accidental addition

Reply:
I think I meant it to be added in `RenovateConfig`.